vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/treegrid/it/TreeGridBasicFeaturesIT.java

```diff
@@ -184,6 +184,19 @@ public void keyboard_navigation() {
         checkLogsForErrors();
     }
 
+    @Test
+    public void keyboard_navigation_row_focus_expand() {
+        getTreeGrid().getCell(0, 0).focus();
+
+        // Enter row focus mode
+        new Actions(getDriver()).sendKeys(Keys.LEFT).perform();
+
+        // Should expand "0 | 0" on right arrow
```

Comment:
Let's also add an assertion for the collapse action. It will ensure the test coverage of this part:

https://github.com/vaadin/flow-components/blob/7ad23fca1789c7e97bc8bd5a3c38d552ce4f7f41/vaadin-grid-flow-parent/vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/gridConnector.js#L505-L538

Reply:
Done